test/module.js

```diff
@@ -298,4 +298,15 @@ describe('unescape of XML ampersands', function () {
         const output = tangram_carto.cartoCssToDrawGroups(ccss, 0);
         assert.ok(!output[0].draw.drawGroup0.color.includes('&amp'));
     });
-});
\ No newline at end of file
+});
+
+describe('Unsupported CartoCSS', function () {
+    it('due to mapnik identifiers should throw an exception', function () {
+        const ccss = `#layer ['mapnik::geometry_type'=2]{
```

Comment:
Even as you mentioned, the regex will work over the carto library generated code and not over user input, I would add another test to validate it works OK with double quotes and spaces, so if for some reason, the generated code changes, we will get some tests broken.